gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GHFSInstrumentation.java

```diff
@@ -0,0 +1,331 @@
+package com.google.cloud.hadoop.fs.gcs;
+
+
+import org.apache.hadoop.fs.FileSystem;
```

Comment:
Use `mvn spotless:apply` to reformat this PR

Reply:
Updated